app/templates/services/service.html

```diff
@@ -0,0 +1,106 @@
+{% extends "_base_page.html" %}
+{% from 'macros/_breadcrumbs.html' import breadcrumb %}
+{% block breadcrumb %}
+  {{ breadcrumb([
+    {
+      "link": url_for(".dashboard"),
+      "text": current_user.supplier_name
+    }
+  ]) }}
+{% endblock %}
+{% block main_content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <header class="page-heading-smaller">
+        {% if service_data.status != 'disabled' %}
+          <p class="context">
+            Edit
+          </p>
+        {% endif %}
+        <h1>
+          {{ service_data['serviceName'] }}
+        </h1>
+      </header>
+        {% if error %}
+        <div class="validation-masthead" aria-labelledby="validation-masthead-heading">
+            <h3 class="validation-masthead-heading" id="validation-masthead-heading">
+                {{ error }}
+            </h3>
+        </div>
+        {% endif %}
+        {% if service_data.status != 'disabled' %}
+      <form action="{{ url_for('.update_service_status', service_id=service_data['id'] ) }}" method="POST">
+        <fieldset class="question">
+          <legend class="question-heading question-heading-with-hint ">
+            Choose service status
+          </legend>
+          <p class="hint">
+            Private services don't appear in search results and don't have a
+            <acronym title="Uniform resource locator">URL</acronym>
+          </p>
+          <label class="selection-button selection-button-boolean">
+            <input type="radio" name="service_status" id="service_status_published" value="public" {{'checked=checked' if service_data['status'] == 'published' }} />
```

Comment:
Attribute values should be in quotes.  I know it works anyway, but still nice to be consistent.  I'll fix this now and then merge it.